features/doctypes/default-doctypes.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Default Doctypes
+  The provided doctypes can be published out of the box.
+
+  # the "@createSchema" annotation provided by Behat creates a temporary SQLite database for testing the API
+  @createSchema
+  Scenario: I can export the default AP
+    Given I am logged in as "admin"
```

Comment:
a scenario should be "Given, _When_ and Then". Given a predicate, when I do an action, then I get this reaction.

Reply:
```
Given I am logged as admin
And I have an etude "Etude 2"
When I access the etude "Etude 2"
Then I should see the etude "Etude 2"
```

Some comments:

- When creating objects, you can give them a label like `"Etude 2"` above, so in a context you can have an array label => object to easily access to them.
- It seems bad phrasing (and it is), the label is used again and again at each sentence. While it could be phrased better, this approach have a big advantage: it makes it stateless. If you don't want to repeat yourself, you would need to keep track of the subject which is a PITA and very error prone. So trading off on the phrasing a bit is a good idea here

- `And I have an etude "Etude 2"`: so you create your object and you store it with the label `Etude 2`
- `When I access the etude "Etude 2"`: for an UI tests, it can be "visit the page X". For a non-UI tests, it can be get the object from the repository
- `Then I should see the etude "Etude 2"` for your UI tests, you can indeed test the response type (HTML, JSON...) and the status code